app/presenters/vendor_api/application_presenter/deferred_offer.rb

```diff
@@ -7,8 +7,15 @@ def schema
         offer: {
           status_before_deferral: application_choice.status_before_deferral,
           offer_deferred_at: application_choice.offer_deferred_at&.iso8601,
+          confirm_deferral_recruitment_cycle_year: confirm_deferral_recruitment_cycle_year,
         },
       },
     })
   end
+
+  def confirm_deferral_recruitment_cycle_year
+    return unless application_choice.offer_deferred?
+
+    application_choice.current_recruitment_cycle_year + 1
```

Comment:
👍🏼 I like this approach because it will work for multiple defer/confirm cycles, if needed.

Reply:
we have other cases, so I will need to update this value as unfortunately its not that clean cut